braindecode/models/biot.py

```diff
@@ -69,10 +69,10 @@ class _ClassificationHead(nn.Sequential):
         (batch, n_outputs)
     """
 
-    def __init__(self, emb_size: int, n_outputs: int):
+    def __init__(self, emb_size: int, n_outputs: int, activation: nn.Module = nn.ELU):
```

Comment:
Docstring needs update. Can you check all docstrings?

Reply:
yes @agramfort!